src/main/kotlin/io/github/anki/anki/controller/dto/DeckDto.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.anki.anki.controller.dto
+
+import jakarta.validation.constraints.NotBlank
+
+data class NewDeckRequest(
+    @field:NotBlank(message = "should not be blank")
+    val name: String?,
+    val description: String?,
+)
+
+data class PatchDeckRequest(
+    val name: String? = null,
+    val description: String? = null,
+)
+
+data class DeckDtoResponse(
```

Comment:
We don't need to respond array of cards inside `Deck`?

Reply:
No, front will get cards with pagination in future